packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -55,6 +57,7 @@ export interface StatusBarEntryAttributes {
     className?: string;
     title?: string;
     style?: object;
+    role?: string;
     onClick?: (e: MouseEvent) => void;
 }
```

Comment:
I believe this interface can be removed completely, I'll open a separate pull-request to do a cleanup.